DeviceAdministration/Web/Web.csproj

```diff
@@ -37,7 +37,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS; SUPPRESS_CSRF_PROTECTION; ALLOW_ANONYMOUS_API</DefineConstants>
```

Comment:
Yes CSRF needs to also be disabled for API to allow cross origin requests. Otherwise it only allows the same origin to make the request with is the Web App. DefineConstants scope is project wide and would only apply to Web in this case.